manifests/reposync-crd.yaml

```diff
@@ -1163,9 +1163,10 @@ spec:
                       false.'
                     type: boolean
                   period:
-                    description: 'period is the time duration between consecutive
-                      syncs. Default: 1hour. Use string to specify this field value,
-                      like "30s", "5m". More details about valid inputs: https://pkg.go.dev/time#ParseDuration.
+                    description: 'period is the time duration that Config Sync waits
+                      before refetching the chart and resyncing. Default: 1 hour.
```

Comment:
Suggest deleting `and resyncing`. If the period is configured to be less than 15s, internally, the reconciler still waits for 15s to resync.

Reply:
Done, took out `and resyncing` and also changed the language below not to reference `syncs`.